executor/runtime/docker/docker.go

```diff
@@ -1704,13 +1710,13 @@ func (r *DockerRuntime) setupPostStartLogDirTiniHandleConnection2(parentCtx cont
 		return fmt.Errorf("error mounting proc pid1 in titus init: %w", err)
 	}
 
-	if r.cfg.UseNewNetworkDriver && c.VPCAllocation().IPV4Address != nil {
+	if r.cfg.UseNewNetworkDriver && c.VPCAllocation() != nil {
```

Comment:
Can UseNewNetworkDriver go away next? Are we not using that everywhere?

Reply:
We apparently have it disabled when we run testing.